examples/multiple-endpoints/main.tf

```diff
@@ -0,0 +1,218 @@
+  http_tcp_listeners = [{
+    port               = 80
+    protocol           = "HTTP"
+    target_group_index = 0
+  }]
+
+  target_groups = [{
+    backend_protocol = "HTTP"
+    backend_port     = 80
+    target_type      = "ip"
+  }]
+
+  tags = local.tags
+}
+
+
+################################################################################
+# Global Accelerator Module
+################################################################################
+
+module "global_accelerator" {
+  source = "../.."
+
+  name = local.name
+
+  # Listeners
+  listeners = {
+    listener_1 = {
+      client_affinity = "SOURCE_IP"
+
+      endpoint_groups = {
+        eu = {
+          endpoint_group_region         = "eu-west-1"
+          health_check_port             = 80
+          health_check_protocol         = "HTTP"
+          health_check_path             = "/"
+          health_check_interval_seconds = 10
+          health_check_timeout_seconds  = 5
+          threshold_count               = 2
+          # unhealthy_threshold_count     = 2
+          traffic_dial_percentage = 100
+
+          # Health checks will show as unhealthy in this example because there
+          # are obviously no healthy instances in the target group
+          endpoint_configuration = [{
+            client_ip_preservation_enabled = true
+            endpoint_id                    = module.alb_eu.lb_arn
+          }]
+
+          port_override = [{
+            endpoint_port = 82
+            listener_port = 80
+          }]
+        }
+        us = {
+          endpoint_group_region         = "us-west-1"
+          health_check_port             = 80
+          health_check_protocol         = "HTTP"
+          health_check_path             = "/"
+          health_check_interval_seconds = 10
+          health_check_timeout_seconds  = 5
+          healthy_threshold_count       = 2
+          unhealthy_threshold_count     = 2
+          traffic_dial_percentage       = 100
+
+          # Health checks will show as unhealthy in this example because there
+          # are obviously no healthy instances in the target group
+          endpoint_configuration = [{
+            client_ip_preservation_enabled = true
+            endpoint_id                    = module.alb_us.lb_arn
+          }]
+
+          port_override = [{
+            endpoint_port = 82
+            listener_port = 80
+          }]
+        }
+      }
+
+      port_ranges = [
+        {
+          from_port = 80
+          to_port   = 81
+        },
+        {
+          from_port = 8080
+          to_port   = 8081
+        }
+      ]
+      protocol = "TCP"
+    }
+  }
+
+  listeners_timeouts = {
+    create = "35m"
+    update = "35m"
+    delete = "35m"
+  }
+
+  endpoint_groups_timeouts = {
+    create = "35m"
+    update = "35m"
+    delete = "35m"
+  }
+
+  tags = local.tags
+}
+
+data "aws_availability_zones" "eu" {
+  provider = aws.eu
+  state    = "available"
+}
+data "aws_availability_zones" "us" {
```

Comment:
Add a new line in between, too. And move them to the "Supporting Resources" block.